ui.frontend/package-lock.json

```diff
@@ -34,18 +34,18 @@
       }
     },
     "node_modules/@aemforms/af-core": {
-      "version": "0.22.52",
```

Comment:
af-core must not be downgraded.

Reply:
Please remove the package-lock changes from this PR